taichi/backends/vulkan/spirv_ir_builder.cpp

```diff
@@ -0,0 +1,940 @@
+#include "taichi/backends/vulkan/spirv_ir_builder.h"
+
+TLANG_NAMESPACE_BEGIN
```

Comment:
nit: Please use `namespace taichi { namespace lang {` instead of this macro. (same for `TLANG_NAMESPACE_END`)